### PR TITLE
Micro optimization to reduce array_merge calls

### PR DESCRIPTION
**Pull request by sidz**
Not a big win but this is a micro optimization will reduce amount of array_merge calls in case `exclude-path` or `exclude-file` options has more than one entry as Finder is doing array merge all the time `notPath`, `notName`, `name` are called.

see https://github.com/symfony/finder/blob/4.4/Finder.php#L286

### CODE REVIEW

File: src/Command/RunCommand.php [hunk 1]
```diff
@@ -227,20 +227,14 @@ private function getCSVOption(InputInterface $input, string $option): array
...
 
-        if (false === is_array($result)) {
-            return array_filter(
-                explode(',', (string) $result),
-                static function ($value) {
-                    return false === empty($value);
-                }
-            );
+        if (!is_array($result)) {
+            return array_filter(explode(',', (string) $result));
```
Comment (sidz) [inline src/Command/RunCommand.php:231]: not passign any callback is identical to 
```
static function ($value) {
    return false === empty($value);
}
```

File: src/Console/Application.php [hunk 2]
```diff
@@ -46,11 +46,11 @@ public function doRun(InputInterface $input, OutputInterface $output): int
 
     public function getLongVersion(): string
     {
-        return trim(sprintf(
+        return sprintf(
```
Comment (sidz) [inline src/Console/Application.php:49]: doesn't make sense to use trim as there are no spaces at the left or right sides of this string

**Comment by sidz** [comment]
Hey @sasezaki, @exussum12, @kubawerlos could you please take a look at these changes when you have a chance?

File: src/Command/RunCommand.php [hunk 3]
```diff
@@ -227,20 +227,14 @@ private function getCSVOption(InputInterface $input, string $option): array
...
         }
 
-
         return $result;
     }
 
-    protected function createFinder(InputInterface $input): PHPFinder
+    private function createFinder(InputInterface $input): PHPFinder
```
Comment (kubawerlos) [inline src/Command/RunCommand.php:237]: Isn't this technically a BC break? Class `RunCommand` is not final, so someone can extend it and call `createFinder`.

Comment (sidz) [reply src/Command/RunCommand.php:237]: Technically yes. But practically I don't think that anybody did this :)

Comment (sidz) [reply src/Command/RunCommand.php]: reverted

**Review by kubawerlos — APPROVED** [review]

**Comment by sidz** [comment]
thanks @kubawerlos, @sasezaki

**Pull request merged**